docs/csharp/toc.yml

```
@@ -1,5 +1,5 @@
 - name: C# Guide
```

Should TOC and other places where we call this Guide be updated as well? We could make similar changes when updating the other landing pages.

```suggestion
- name: C# documentation
```